gpu/tasks/nvidia_ctoolkit.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Install NVIDIA Container Toolkit and Runtime
+  ansible.builtin.apt:
+    name: 
+      - nvidia-container-toolkit
+      - nvidia-container-runtime
+    state: present
+    update_cache: yes
+
+- name: Check if NVIDIA runtime configuration file exists
+  ansible.builtin.stat:
+    path: /etc/nvidia-container-runtime/config.toml
+  register: nvidia_config_file
+
+- name: Enable NVIDIA visible devices as volume mounts
+  ansible.builtin.lineinfile:
+    path: /etc/nvidia-container-runtime/config.toml
+    regexp: '#accept-nvidia-visible-devices-as-volume-mounts = false'
+    line: 'accept-nvidia-visible-devices-as-volume-mounts = true'
+    backrefs: yes
+  when: nvidia_config_file.stat.exists
+
+- name: Disable NVIDIA visible devices envvar when unprivileged
+  ansible.builtin.lineinfile:
+    path: /etc/nvidia-container-runtime/config.toml
+    regexp: '#accept-nvidia-visible-devices-envvar-when-unprivileged = true'
+    line: 'accept-nvidia-visible-devices-envvar-when-unprivileged = false'
+    backrefs: yes
+  when: nvidia_config_file.stat.exists
+
+- name: Create configuration to disable automatic updates
```

Comment:
These tasks:
- `Create configuration to disable automatic updates`
- `Remove unattended-upgrades package`
 
belong to a more general role related to server-level configuration rather than a GPU-specific task.
Say it could be a part of `common/tasks/main.yml` (instead of `gpu`)
or maybe `os_config/tasks/disable_auto_updates.yml`, and include it in `roles/os_config/tasks/main.yml` (e.g.: `- import_tasks: disable_auto_updates.yml`)

Up to you. (We can always change it later if we want to).

There can be more such configs, including sysctl bits/etc.